### PR TITLE
fix(AAE-11372): Create alpha releases for alfresco-process-application-deployment

### PR DESCRIPTION
**Pull request by igdianov**
Fixes: https://alfresco.atlassian.net/browse/AAE-11372

Depends on
- https://github.com/Alfresco/alfresco-build-tools/pull/186
- https://github.com/Alfresco/alfresco-process-releases/pull/115

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -69,29 +71,67 @@ jobs:
...
-      - name: Get commit message
-        id: get-commit-message
-        shell: bash
-        run: |
-          MSG=$(git log --pretty=format:%s --max-count=1)
-          echo "message=$MSG" >> $GITHUB_OUTPUT
+  release:
+    runs-on: ubuntu-latest
+    needs: build
+    if: github.event_name == 'push'
```
Comment (atchertchian) [inline .github/workflows/main.yml]: Please remove this check so that we can check most of this logic in PRs

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -69,29 +71,67 @@ jobs:
...
+          version: ${{ steps.calculate-next-internal-version.outputs.next-prerelease }}
+          chart-dir: ${{ env.CHART_DIR }}
+          helm-repository: ${{ env.HELM_REPO }}
+          helm-repository-branch: ${{ env.HELM_REPO_BRANCH }}
+          helm-repository-subfolder: ${{ env.HELM_REPO_SUBFOLDER }}
+          helm-repository-base-url: ${{ env.HELM_REPO_BASE_URL }}
+          helm-repository-token: ${{ secrets.BOT_GITHUB_TOKEN }}
           git-username: ${{ secrets.BOT_GITHUB_USERNAME }}
-          git-message: ${{ steps.get-commit-message.outputs.message }}
-          do-push: ${{ github.event_name == 'push' }}
```
Comment (atchertchian) [inline .github/workflows/main.yml:90]: Please keep that input as I added it on the helm-release-and-publish action too

File: .github/workflows/main.yml (hunk 1 above)
Comment (igdianov) [reply .github/workflows/main.yml]: Fixed

File: .github/workflows/main.yml (hunk 2 above)
Comment (igdianov) [reply .github/workflows/main.yml:90]: fixed

**Review by erdemedeiros — APPROVED** [review]
LGTM!

**Pull request merged**